bin/action_subscriber

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env ruby
 
-require 'active_support'
-require 'active_support/core_ext'
 require 'thor'
+require 'action_subscriber'
```

Comment:
similar to the thor issue, this was working previously because rails had already loaded our gem, but I thought it was worth fixing so that the binstub just loads thor and the gem itself to make it less dependent on the behavior of what happens when we require `config/environment.rb`